eslint.config.js

```diff
@@ -0,0 +1,40 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+import pluginReact from "eslint-plugin-react";
+
+export default [
+  {
+    files: ["src/*.{js,mjs,cjs,ts,jsx,tsx}"],
+  },
+  {
+    languageOptions:
+    {
+      globals: { ...globals.browser, ...globals.node }
+    },
+  },
+  {
+    settings: {
+      react: {
+        version: '18.2',
+      },
+    },
+  },
+  {
+    ignores: ["src/components/ui/*", "src/components/svgs/*", "build/"],
```

Comment:
why the components/ui?

Reply:
Because the shadcn components generate a lot of errors and we didn't create those, so I don't think we should bother fixing what the linter is complaining about (mainly the typescript ones).

Reply:
ok, I see. Makes sense